src/find/azext_find/custom.py

```diff
@@ -66,6 +67,10 @@ def process_query(cli_command):
             telemetry_core.add_extension_event(EXTENSION_NAME, properties)
 
 
+def style_message(msg):
+    return colorama.Style.BRIGHT + msg + colorama.Style.RESET_ALL
```

Comment:
Need to give consideration to terminals that won't display color. This will appear as gibberish.

Reply:
That's right, the new commit should address your point.